.changelog/2787.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:security
+Add NET_BIND_SERVICE capability to restricted security context used for consul-dataplane
```

Comment:
@david-yu do you have thoughts on how this appears in the changelog? I was torn between `improvement` and `security` as the not category and wasn't sure if it made sense to include any more details here or a link to the related consul-dataplane PR which requires this change.

Reply:
I believe this is more of an improvement than a security change which is typically reserved for addressing CVEs.

Reply:
Updated